### PR TITLE
pkg: remove capnslog

### PR DESCRIPTION
**Pull request by jingyih**
Remove dependency of `capnslog` in `pkg` package.

Related: #11426

### CODE REVIEW

File: etcdmain/config.go [hunk 1]
```diff
@@ -321,7 +321,7 @@ func (cfg *config) configFromCmdLine() error {
 		}
 	}
 
-	err := flags.SetFlagsFromEnv("ETCD", cfg.cf.flagSet)
+	err := flags.SetFlagsFromEnv(cfg.ec.GetLogger(), "ETCD", cfg.cf.flagSet)
```
Comment (jingyih) [inline etcdmain/config.go]: Note: At this point, I don't think user-configured zap logger is properly setup yet. Should we use zap.NewProduction() or maybe a different logger?

Comment (gyuho) [reply etcdmain/config.go]: `zap.NewProduction` should be fine.

**Comment by xiang90** [comment]
@jingyih can you resolve the conflict in the changlog? thanks.

**Comment by jingyih** [comment]
> @jingyih can you resolve the conflict in the changlog? thanks.

@xiang90 Rebased and resolved.

**Pull request merged**